ubiquity/dell-bootstrap.py

```diff
@@ -998,6 +1069,11 @@ def build_rp(self, cushion=600):
                     uuid = item.split('=')[1]
                     break
 
+        with misc.raised_privileges():
+            if self.rec_type=='dhc':
+                file=open('/mnt' + '/.dhc_flag',"w")
```

Comment:
rather than two lines like this, can you please follow the style and use `with open ` instead and also use `os.path.join` instead?

Reply:
https://www.pythonforbeginners.com/files/with-statement-in-python for an example if you need it.

Reply:
Thank you for your great recommend, I think it is great learn python for a beginner. And I will follow the style.